content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -1035,6 +1035,13 @@ kube-apiserver [flags]
       <td></td><td style="line-height: 130%; word-wrap: break-word;">Watch cache size settings for some resources (pods, nodes, etc.), comma separated. The individual setting format: resource[.group]#size, where resource is lowercase plural (no version), group is omitted for resources of apiVersion v1 (the legacy core API) and included for others, and size is a number. It takes effect when watch-cache is enabled. Some resources (replicationcontrollers, endpoints, nodes, pods, services, apiservices.apiregistration.k8s.io) have system defaults set by heuristics, others default to default-watch-cache-size</td>
     </tr>
 
+    <tr>
```

Comment:
This is generated documentation. You may not need to change that in this pull request.

Reply:
> This is generated documentation. You may not need to change that in this pull request.

Oh? But how to document the new flags?